fix Docker docu

```bash
apfeltasche:~ mschleichardt$ docker pull sphereio/sunrise
Using default tag: latest
latest: Pulling from sphereio/sunrise

3b59c795b346: Pull complete
7fbafd6dc0a3: Pull complete
d1a3521d63d3: Pull complete
2ef145c1bb2f: Pull complete
69b0edc1167e: Pull complete
b860b6e6a25b: Pull complete
f5bd1e4b8254: Pull complete
82bb16225afa: Pull complete
63a8ce773a2c: Pull complete
61674e153398: Pull complete
4b5a316bf26b: Pull complete
6c8ae48cc836: Pull complete
358229e08677: Pull complete
26a0665ca54f: Pull complete
509ecbf94b66: Pull complete
a2481654fd1a: Pull complete
864bc72ef1c4: Pull complete
cf4d70ec4030: Pull complete
745f96140768: Pull complete
7a25ca0caaae: Pull complete
2ad91ee96f65: Pull complete
Digest: sha256:4a7b037e70e97a057d889fcd86bb2ebd443e72fa1adaaa483ac9d381a3e30179
Status: Downloaded newer image for sphereio/sunrise:latest

apfeltasche:~ mschleichardt$ docker run -it --rm -p 9000:9000 sphereio/sunrise
Oops, cannot start the server.
Configuration error: Configuration error[prod.conf @ file:/opt/docker/conf/prod.conf: 7: Could not resolve substitution to a value: ${CTP_PROJECT_KEY}]
	at play.api.Configuration$.configError(Configuration.scala:178)
	at play.api.Configuration$.load(Configuration.scala:103)
	at play.core.server.ProdServerStart$.readServerConfigSettings(ProdServerStart.scala:81)
	at play.core.server.ProdServerStart$.start(ProdServerStart.scala:42)
	at play.core.server.ProdServerStart$.main(ProdServerStart.scala:27)
	at play.core.server.ProdServerStart.main(ProdServerStart.scala)
Caused by: com.typesafe.config.ConfigException$UnresolvedSubstitution: prod.conf @ file:/opt/docker/conf/prod.conf: 7: Could not resolve substitution to a value: ${CTP_PROJECT_KEY}
	at com.typesafe.config.impl.ConfigReference.resolveSubstitutions(ConfigReference.java:108)
	at com.typesafe.config.impl.ResolveContext.realResolve(ResolveContext.java:179)
	at com.typesafe.config.impl.ResolveContext.resolve(ResolveContext.java:142)
	at com.typesafe.config.impl.SimpleConfigObject$ResolveModifier.modifyChildMayThrow(SimpleConfigObject.java:379)
	at com.typesafe.config.impl.SimpleConfigObject.modifyMayThrow(SimpleConfigObject.java:312)
	at com.typesafe.config.impl.SimpleConfigObject.resolveSubstitutions(SimpleConfigObject.java:398)
	at com.typesafe.config.impl.ResolveContext.realResolve(ResolveContext.java:179)
	at com.typesafe.config.impl.ResolveContext.resolve(ResolveContext.java:142)
	at com.typesafe.config.impl.SimpleConfigObject$ResolveModifier.modifyChildMayThrow(SimpleConfigObject.java:379)
	at com.typesafe.config.impl.SimpleConfigObject.modifyMayThrow(SimpleConfigObject.java:312)
	at com.typesafe.config.impl.SimpleConfigObject.resolveSubstitutions(SimpleConfigObject.java:398)
	at com.typesafe.config.impl.ResolveContext.realResolve(ResolveContext.java:179)
	at com.typesafe.config.impl.ResolveContext.resolve(ResolveContext.java:142)
	at com.typesafe.config.impl.ResolveContext.resolve(ResolveContext.java:231)
	at com.typesafe.config.impl.SimpleConfig.resolveWith(SimpleConfig.java:74)
	at com.typesafe.config.impl.SimpleConfig.resolve(SimpleConfig.java:64)
	at com.typesafe.config.impl.SimpleConfig.resolve(SimpleConfig.java:59)
	at com.typesafe.config.impl.SimpleConfig.resolve(SimpleConfig.java:37)
	at play.api.Configuration$.load(Configuration.scala:99)
	... 4 more
```

--- a/README.md
+++ b/README.md
@@ -52,7 +52,7 @@ For an easy and fast deployment of your application we recommend [heroku](https:
 You can also use a ready-to-run docker container.
 
 ```bash
-$ docker run -it --rm -p 9000:9000 sphereio/sunrise
+$ docker run -it --rm -p 9000:9000 -e CTP_PROJECT_KEY=YOUR_PROJECT_KEY -e CTP_CLIENT_ID=YOUR_CLIENT_ID -e CTP_CLIENT_SECRET=YOUR_CLIENT_SECRET -e APPLICATION_SECRET=YOUR_PLAY_APPLICATION_SECRET sphereio/sunrise
 ```
 
 ## Related projects

